### PR TITLE
OSDOCS-6880: Adding 64-bit ARM GCP instance types to documentation

### PR DESCRIPTION
**Pull request by kelbrown20**
**Jira story:**  [OSDOCS-6880](https://issues.redhat.com/browse/OSDOCS-6880)
Subtask of [OSDOCS-6319](https://issues.redhat.com/browse/OSDOCS-6319) [Documentation enablement for GCP on ARM]

**Description:** Adding supported ARM64 GCP instance types to installer documentation. Will also use this file for OCP documentation

### CODE REVIEW

**Comment by kelbrown20** [comment]
@r4f4 @AnnaZivkovic Hi Rafael Hi Tori! Can you take a look at this PR on the ARM side when you get the chance? Thank you!

**Comment by r4f4** [comment]
@kelbrown20 We need QE to validate those instance types. During development I hadn't even seen T2D and N2D types.

**Comment by r4f4** [comment]
From what I can see, T2D is `x86_64`, not `arm64`.

**Comment by kelbrown20** [comment]
Ah gotcha! I might have made this PR a bit early lol. Will confirm with QE when the instance types are validated. Thank you!

**Comment by r4f4** [comment]
@kelbrown20 We'll have to include in the docs that arm64 instances are [only available in select regions/zones](https://cloud.google.com/compute/docs/instances/create-arm-vm-instance#available_regions). There is a table [here]( https://cloud.google.com/compute/docs/regions-zones#available)

**Comment by kelbrown20** [comment]
Thank you @r4f4! Currently have a WIP PR for the GCP doc updates, still adding some more details (https://github.com/openshift/openshift-docs/pull/62253)

**Comment by r4f4** [comment]
@kelbrown20 Is this still needed after https://github.com/openshift/installer/pull/7381 ?

**Comment by r4f4** [comment]
@kelbrown20 ok, can you comment on https://github.com/openshift/installer/pull/7381?

File: docs/user/gcp/tested_instance_types_arm.md [hunk 1]
```diff
@@ -0,0 +1 @@
+* `T2A`
```
Comment (jianli-wei) [inline docs/user/gcp/tested_instance_types_arm.md:1]: Suggest to name as the same as [Google doc](https://cloud.google.com/compute/docs/general-purpose-machines#t2a_machines), i.e. `Tau T2A`, thanks!

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: docs/user/gcp/tested_instance_types_arm.md (hunk 1 above)
Comment (kelbrown20) [reply docs/user/gcp/tested_instance_types_arm.md]: Just updated, thank you!

**Comment by kelbrown20** [comment]
@r4f4 PTAL, thank you!

**Comment by aleskandro** [comment]
hi @kelbrown20 lgtm!

**Pull request merged**